src/controller/config.js

```diff
@@ -155,11 +165,39 @@ const drawNewImage = async (img, x, y) => {
     
 }
 
-const drawNewText = (txt, currCoordinates) => {
+const drawTextBackground = async (ctx, txt, x, y, font, padding) => {
+    return new Promise((resolve) => {
```

Comment:
wrapped in promise so it can be drawn first. In testing I found that the text can be drawn first and the background can be placed on top